erpnext/hr/doctype/leave_application/leave_application.py

```diff
@@ -340,6 +402,36 @@ def get_approved_leaves_for_period(employee, leave_type, from_date, to_date):
 
 	return leave_days
 
+def get_total_approved_leaves_for_period(employee, from_date, to_date):
```

Comment:
Why rewriting the same code twice?

It is same as `get_approved_leaves_for_period`.  Make leave_type args optional.